spec/ruby-lint/analysis/argument_amount_spec.rb

```diff
@@ -108,11 +109,13 @@ def initialize(name)
 
     first.line.should    == 6
     first.column.should  == 1
-    first.message.should == 'wrong number of arguments (expected 1 but got 0)'
+    first.message.should == 'wrong number of arguments for \'initialize\' ' \
+      '(expected 1 but got 0)'
 
     second.line.should    == 7
     second.column.should  == 1
-    second.message.should == 'wrong number of arguments (expected 1 but got 2)'
+    second.message.should == 'wrong number of arguments for \'initialize\' ' \
```

Comment:
For these tests we can just use double quotes, this removes the need for escaping the single quotes.